presto-main/src/main/java/io/prestosql/dispatcher/QueuedStatementResource.java

```diff
@@ -158,13 +158,13 @@ public void stop()
     }
 
     /**
-     * Receives POST requests that tries to create new queries to be executed on server.
+     * Receives POST requests that try to create new queries to be executed on the server.
      *
-     * @param statement a sql command
-     * @param servletRequest an object that provides information about the http request
+     * @param statement a SQL command
+     * @param servletRequest an object that provides information about the HTTP request
      * @param httpHeaders an object containing information about the request headers
-     * @param uriInfo an object containing metadata about the uri of endpoint
-     * @return the information about query that was created
+     * @param uriInfo an object containing metadata about the URI of endpoint
+     * @return the information about the query that was created
```

Comment:
The word "Information" seems too generic. What is this information? The query metadata?
Let's try not to be too generic.

Reply:
changed to **the result of the processed query**